Task 2.py

```diff
@@ -0,0 +1,18 @@
+# Напишите 2 функции:
+# Которая запрашивает у пользователя число и выводит его квадрат
+# Которая запрашивает у пользователя число и определяет, является ли оно четным или нечетным
+
+def num_expo(x):
+    return x * x
+
+
+def even_odd(y):
+    if y % 2 == 0:
+        return "Even"
+    else:
+        return "Odd"
```

Comment:
Rename function to: is_even
```suggestion
    if y % 2 == 0:
        return True
    return False
```